src/types.ts

```diff
@@ -11,7 +28,8 @@ export interface ParseSettings {
 }
 
 export interface JsonToken {
-  type: any;
-  value: any;
+  type: JsonTokenTypes;
+  value: string;
+  decoded?: string;
```

Comment:
Would it be like this if made more specific with types?
```suggestion
  decoded?: string | null;
```